config/areas/languages/buttons.php

```diff
@@ -1,35 +1,21 @@
 <?php
 
 use Kirby\Cms\Language;
+use Kirby\Panel\Ui\Buttons\LanguageAddButton;
+use Kirby\Panel\Ui\Buttons\LanguageDeleteButton;
+use Kirby\Panel\Ui\Buttons\LanguageSettingsButton;
 use Kirby\Panel\Ui\Buttons\PreviewButton;
-use Kirby\Panel\Ui\Buttons\ViewButton;
-use Kirby\Toolkit\I18n;
 
 return [
-	'languages.add' => function () {
-		return new ViewButton(
-			dialog: 'languages/create',
-			icon: 'add',
-			text: I18n::translate('language.create'),
-		);
-	},
-	'language.preview' => function (Language $language) {
-		return new PreviewButton(link: $language->url());
-	},
-	'language.settings' => function (Language $language) {
-		return new ViewButton(
-			dialog: 'languages/' . $language->id() . '/update',
-			icon: 'cog',
-			title: I18n::translate('settings'),
-		);
-	},
+	'languages.add' => fn () =>
+		new LanguageAddButton(),
+	'language.preview' => fn (Language $language) =>
+		new PreviewButton(link: $language->url()),
+	'language.settings' => fn (Language $language) =>
+		new LanguageSettingsButton($language),
 	'language.remove' => function (Language $language) {
 		if ($language->isDeletable() === true) {
-			return new ViewButton(
-				dialog: 'languages/' . $language->id() . '/delete',
-				icon: 'trash',
-				title: I18n::translate('delete'),
-			);
+			return new LanguageDeleteButton($language);
```

Comment:
`Reasoning`: Why did you go for `LanguageDeleteButton` instead of `LanguageRemoveButton`.

Reply:
I guess to be more in line with the dialog name and i18n string. But could be either way.

Reply:
Oh you are right. I thought the dialog was also called language.remove. Maybe we can update the button keys instead to be in line with the dialog route keys.

Reply:
Renamed it and did the same then with `users.create` instead `users.add`